README.md

```diff
@@ -5,7 +5,7 @@ This holds *all* of our [best practices](about.md).
 
 ## Git
 
-1. [Setup](git/setup.md)
+1. [Setup](git/00.setup.md)
```

Comment:
I'd probably not do this because if you add a document in the middle of the list you have to update all the others.